wled00/pin_manager.cpp

```diff
@@ -243,7 +243,6 @@ bool PinManagerClass::isPinOk(byte gpio, bool output) const
   #if defined(CONFIG_IDF_TARGET_ESP32C3)
     // strapping pins: 2, 8, & 9
     if (gpio > 11 && gpio < 18) return false;     // 11-17 SPI FLASH
-    if (gpio > 17 && gpio < 20) return false;     // 18-19 USB-JTAG
```

Comment:
This should only be disabled in case of ARDUINO_USB_CDC_ON_BOOT=0
Otherwise debugging over native USB will not work any more (purpose of ARDUINO_USB_CDC_ON_BOOT=1)